dynamiq/nodes/tools/http_api_call.py

```diff
@@ -32,7 +53,7 @@ class HttpApiCall(ConnectionNode):
         response_type(ResponseType|str): The type of response content.
     """
 
-    name: str = "The name of the API call tool"
+    name: str = "Api call tool"
```

Comment:
Why some nodes name using human like words in naming and few nodes renamed to `tool-name` style?